### PR TITLE
109 add GitHub, Netlify to footer

### LINKED ISSUE #109: Add link to GitHub source in the footer
Let's link to this repo in the footer so people can easily find the source and contribute.

### PR DESCRIPTION
**Pull request by ann-kilzer**
## Part of #108 

Updates the footer to include links to GitHub and Netlify

TODO:
- [x] Make the footer smaller on mobile, perhaps use icons instead of text?

### How to test

Check in the Netlify Preview, ensure the links work

### Screenshots

COVID-19_Vaccination_Info

COVID-19_Vaccination_Info

### CODE REVIEW

**Comment by theyokohamalife** [comment]
I'd like to change the `Buy me a Ko-Fi` to just simply `Donate`.

File: components/SiteFooter.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <a href="https://ko-fi.com/theyokohamalife" target="_blank">
+              {{ $t("footer.donate") }}</a
+            >
+          </v-col>
+          <v-col id="github">
+            <a
+              href="https://github.com/ourjapanlife/findadoc-frontend"
+              target="_blank"
+            >
+              {{ $t("footer.github") }}</a
```
Comment (Ermish) [inline components/SiteFooter.vue:19]: We could use a github icon here!

**Review by Ermish — APPROVED** [review]
LGTM!

File: components/SiteFooter.vue (hunk 1 above)
Comment (ann-kilzer) [reply components/SiteFooter.vue:18]: Yeah it might be fun to try it. :octocat:

**Review by theyokohamalife — APPROVED** [review]
LGTM!

**Pull request merged**